service/views.py

```diff
@@ -13,11 +14,36 @@ def get_queryset(self):
         return super().get_queryset().filter(client=self.request.user)
 
 
-class TherapistMassageAPIView(ListAPIView):
+class TherapistMassageListAPIView(ListAPIView):
+    """ Get list of all massages for current logged in therapist """
     queryset = Massage.objects.all()
     serializer_class = MassageSerializer
     permission_classes = (IsAuthenticated,)
 
     def get_queryset(self):
         return super().get_queryset().filter(therapist=self.request.user)
 
+
+class TherapistScheduleAPIView(RetrieveAPIView):
+    """ Get schedule for any Therapist """
+    queryset = Schedule.objects.all()
+    serializer_class = ScheduleSerializer
+    permission_classes = (IsAuthenticated,)
+
+    def get_queryset(self):
+        pk = self.kwargs['pk']
+        return super().get_queryset().filter(therapist__id=pk)
+
+    def get_object(self):
+        return super().get_object()
+
+
+class TherapistSlotListAPIView(ListAPIView):
```

Comment:
Do you have pagination? 
If not, please add pagination for all lists.
(It should be one line in code)